changelog/DIlRBhW6S1aQ6Rl63gLxog.md

```diff
@@ -0,0 +1,4 @@
+audience: users
+level: minor
```

Comment:
I think this needs to be major, because D2G now uses docker by default for non-privileged tasks, whereas before it used podman.